api/PayrollProcessor.Core.Domain/PayrollProcessor.Core.Domain.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Ardalis.GuardClauses" Version="1.5.0" />
+    <PackageReference Include="LanguageExt.Core" Version="3.4.14" />
```

Comment:
I'm not sure how I feel about having the Domain project depend on multiple packages.

One one hand, they are all just types/config and not services or external / volatile dependencies.

On the other hand I wonder if a `Core.Processes` project would be a good idea for the CQRS types, `IDateProvider`, ect...

I don't think we need to do anything about it now, but it's interesting to think about 🤔.